src/Api/ErrorParser/JsonParserTrait.php

```diff
@@ -15,12 +15,26 @@ trait JsonParserTrait
     private function genericHandler(ResponseInterface $response)
     {
         $code = (string) $response->getStatusCode();
+        if ($this->api &&
```

Comment:
Could we have these operators precede the expression they pertain to? i.e. 

```
(<expression1>
  && <expression2>
  && <expression3>
) {

Reply:
Yes for sure